test/AutoInstall_test.rb

```diff
@@ -1,37 +1,61 @@
 #!/usr/bin/env rspec
 
-require_relative "test_helper"
+require_relative 'test_helper'
+require 'autoinstall/autoinst_issues'
+require 'autoinstall/autoinst_issues_presenter'
+require 'autoinstall/dialogs/question'
 
-Yast.import "AutoInstall"
-
-describe "Yast::AutoInstall" do
+Yast.import 'AutoInstall'
+Yast.import 'UI'
 
+describe 'Yast::AutoInstall' do
   subject { Yast::AutoInstall }
 
-  describe "#pkg_gpg_check" do
-    let(:data) { { "CheckPackageResult" => Yast::PkgGpgCheckHandler::CHK_OK } }
+  describe '#pkg_gpg_check' do
```

Comment:
There is no Rubocop in this module... yet :) But, please, remember that we always use double quotes (").

Reply:
Yes, I have also seen this. But with the latest yast rake packages and "rubocop --auto-correct" it has been  set from " to '. So I have assumed that this will be the default. Is there an error in the rubocop definition file ?